### PR TITLE
Use tiup for BR, Lightning and Dumpling.

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

- Use `tiup br ...` instead of `./br ...` as TiUP offers version management etc.
- Same or `./dumpling` and `./tidb-lightning`
- Use `aws s3 rm...` instead of `rm...` for S3 resources

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by BornChanger** [comment]
@Benjamin2037 please take a look at the lightning part change.

**Review by BornChanger — APPROVED** [review]
br part lgtm

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: br/br-pitr-manual.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: TiDB Log Backup and PITR Command Manual
-summary: TiDB Log Backup and PITR Command Manual describes commands for log backup and point-in-time recovery. Use `br log` command to start, pause, resume, stop, truncate, and query log backup tasks. Specify parameters like `start-ts`, `task-name`, `--storage`, and `--pd` for log backup. Use `br log metadata` to view backup metadata and `br restore point` for PITR. Be cautious when pausing and resuming backup tasks.
+summary: TiDB Log Backup and PITR Command Manual describes commands for log backup and point-in-time recovery. Use `tiup br log` command to start, pause, resume, stop, truncate, and query log backup tasks. Specify parameters like `start-ts`, `task-name`, `--storage`, and `--pd` for log backup. Use `tiup br log metadata` to view backup metadata and `tiup br restore point` for PITR. Be cautious when pausing and resuming backup tasks.
```
Comment (Oreoxmt) [inline br/br-pitr-manual.md:3]:
```suggestion
summary: Introduce the commands used in TiDB log backup and point-in-time recovery (PITR).
```

**Review by Oreoxmt — APPROVED** [review]

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -84,14 +84,14 @@ This section provides some URI examples by using `external` as the `host` parame
...
 --storage "azure://external/backup-20220915?account-name=${account-name}&account-key=${account-key}"
 ```
 
 **Restore the `test` database from snapshot backup data in Azure Blob Storage**
 
 ```shell
-./br restore db --db test -u "${PD_IP}:2379" \
+tiup br restore db --db test -u "${PD_IP}:2379" \
 --storage "azure://external/backup-20220915account-name=${account-name}&account-key=${account-key}"
 ```
```
Comment (Oreoxmt) [inline br/backup-and-restore-storages.md:97]: @dveeden Could you please confirm if Line 131 needs to be adjusted to "tiup br backup full --pd "${PD_IP}:2379"
https://github.com/pingcap/docs/blob/29a6eda38422efdf8c9c1f104e6ee53eb377341e/br/backup-and-restore-storages.md?plain=1#L131

File: br/br-incremental-guide.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Incremental data of a TiDB cluster is differentiated data between the starting s
 
 ## Back up incremental data
 
-To back up incremental data, run the `br backup` command with **the last backup timestamp** `--lastbackupts` specified. In this way, br command-line tool automatically backs up incremental data generated between `lastbackupts` and the current time. To get `--lastbackupts`, run the `validate` command. The following is an example:
+To back up incremental data, run the `tip br backup` command with **the last backup timestamp** `--lastbackupts` specified. In this way, br command-line tool automatically backs up incremental data generated between `lastbackupts` and the current time. To get `--lastbackupts`, run the `validate` command. The following is an example:
```
Comment (Oreoxmt) [inline br/br-incremental-guide.md:16]:
```suggestion
To back up incremental data, run the `tiup br backup` command with **the last backup timestamp** `--lastbackupts` specified. In this way, br command-line tool automatically backs up incremental data generated between `lastbackupts` and the current time. To get `--lastbackupts`, run the `validate` command. The following is an example:
```

**Review by Oreoxmt — APPROVED** [review]

**Comment by Oreoxmt** [comment]
@dveeden https://github.com/dveeden/docs/pull/3 PTAL

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**